### PR TITLE
Updated doc for command line options

### PR DESCRIPTION
**Pull request by omichel**
Refer to https://github.com/omichel/webots/pull/3756

:+1:

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

File: guide/starting-webots.md [hunk 1]
```diff
@@ -101,27 +110,20 @@ to redirect the controllers output to a file or to pipe it to a shell command.
...
+| port               | 1234          | The port on which the streaming server is open. |
+| monitorActivity    |               | Print a dot '.' on stdout every 5 seconds.      |
+| disableTextStreams |               | Disable the streaming of stdout and stderr.     |
+
 
 %end
 
-For example, the following command will start Webots with the streaming server
-enabled on the TCP port '1234': `webots --start-streaming-server="port:1234"`
+For example, the following command will start Webots with the streaming server enabled on the TCP port '1234' and will disable the streaming of stdout and stderr: `webots --start-streaming-server="port:1234;disableTextStreams"`
```
Comment (stefaniapedrazzi) [inline guide/starting-webots.md]: ``--start-streaming-server`` in the example is not updated

**Comment by omichel** [comment]
:+1: (I fixed the example).

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**